micropython/uhttpd/http_file_handler.py

```diff
@@ -211,23 +214,16 @@ def components(self, path):
             f, path.strip('/').split('/')
         )
 
-    def filter(self, f, el):
-        ret = []
-        for e in el:
-            if f(e):
-                ret.append(e)
-        return ret
+    @staticmethod
+    def filter(f, el):
+        return [e for e in el if f(e)]
 
     @staticmethod
     def create_response(code, content_type, length, body):
-        return {
-            'code': code,
-            'headers': {
-                'content-type': content_type,
-                'content-length': length
-            },
-            'body': body
-        }
+        return {'code': code,
```

Comment:
Sorry, I prefer the indentation style I am using.  I realize it is non-standard, but it is regular, which why I prefer it.  (Consistency is better than ad-hoc)